File: engine/model/src/main/scala/tools/packet/showitemgaininchat/ShowOwnRecovery.scala
package tools.packet.showitemgaininchat

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowOwnRecovery(private var _amount: Byte) extends PacketInput {
  def amount: Byte = _amount

  override def opcode(): SendOpcode = SendOpcode.SHOW_ITEM_GAIN_INCHAT
}

File: engine/engine-model/src/main/java/tools/packet/showitemgaininchat/ShowOwnRecovery.java
package tools.packet.showitemgaininchat;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowOwnRecovery(Byte amount) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_ITEM_GAIN_IN_CHAT;
   }
}